pkg/aggregator/internal/tags/store.go

```diff
@@ -206,5 +241,7 @@ func (s *entryStats) visit(e *Entry, r uint64) {
 		s.maxSize = n
 	}
 	s.sumSize += n
+	s.sumSizeBytes += e.SizeInBytes() + e.tags.SizeInBytes()
```

Comment:
Doesn't `e.SizeInBytes` already include `e.tags.SizeInBytes`?

Reply:
It does :(